modules/my_health/app/controllers/my_health/v1/prescriptions_controller.rb

```diff
@@ -189,8 +180,7 @@ def resource_data_modifications(resource)
                              # TODO: remove this line when PF and PD are allowed on va.gov
                              resource.records = remove_pf_pd(resource.data)
                            end
-        resource.records = group_prescriptions(resource.data) if display_grouping
-        resource.records = filter_non_va_meds(resource.data)
```

Comment:
Should we be removing the filter_non_va_meds here? Feels like this is a logic change unrelated to removing the feature toggle.